### PR TITLE
chore: update manage page

### PR DESCRIPTION
**Pull request by aliakseilatyp**
Thanks for submitting a PR! Please check the boxes below:

- [ ] I have run [`pre-commit`](https://docs.flagsmith.com/platform/contributing#pre-commit) to check linting
- [ ] I have filled in the "Changes" section below?
- [ ] I have filled in the "How did you test this code" section below?
- [ ] I have used a [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/) title for this Pull Request

## Changes

- updated manage page
- payments modal replaced with redirect to manage page
- updated panel styles in dark theme

## How did you test this code?

_Please describe._

### CODE REVIEW

**Review by kyle-ssg — COMMENTED** [review]
The payment buttons don't work on this branch. They work on other preview branches e.g. https://flagsmith-frontend-preview-git-chore-updatesid-f823e3-flagsmith.vercel.app

**Review by kyle-ssg — COMMENTED** [review]
image

Border radius/column widths on graph

File: frontend/web/components/App.js [hunk 1]
```diff
@@ -334,9 +333,10 @@ const App = class extends Component {
                                     Utils.getFlagsmithHasFeature(
                                       'payments_enabled',
                                     ) && (
-                                      <a
-                                        href='#'
-                                        className='cursor-pointer nav-link p-2'
+                                      <NavLink
+                                        to='/organisation-settings'
```
Comment (kyle-ssg) [inline frontend/web/components/App.js]: Note: Need to check what a non-admin would see on /organisation-settings

**Review by kyle-ssg — COMMENTED** [review]
Note: Need to check what non-org admins see in this scenario

File: frontend/web/components/AdminAPIKeys.js
Comment (kyle-ssg) [inline frontend/web/components/AdminAPIKeys.js:1]: image

image

Comment (kyle-ssg) [reply frontend/web/components/AdminAPIKeys.js:1]: image

File: frontend/web/components/DateSelect.js
Comment (kyle-ssg) [inline frontend/web/components/DateSelect.js:1]: image

**Comment by kyle-ssg** [comment]
Users without organisation admin privileges may reach the organisation settings page from an upsell CTA. In this scenario the UI will show the following 
image

**Comment by matthewelwell** [comment]
A couple of issues I found testing this: 

1. These 2 buttons do nothing. Probably they are both trying to do the same thing. 

Screenshot 2023-11-20 at 12 53 23



2. This is minor, but I'm not sure we should show the 'Manage Invoices' button for users on the Free Plan? 

Screenshot 2023-11-20 at 12 57 45

**Comment by kyle-ssg** [comment]
> A couple of issues I found testing this:
> 
> 1. These 2 buttons do nothing. Probably they are both trying to do the same thing.
> 
> Screenshot 2023-11-20 at 12 53 23
> 2. This is minor, but I'm not sure we should show the 'Manage Invoices' button for users on the Free Plan?
> 
> Screenshot 2023-11-20 at 12 57 45

They open crisp, crisp just isn't on the preview urls. I can confirm it works.

**Review by matthewelwell — APPROVED** [review]

**Pull request merged**